notebooks/src/el-paso-workflow.ipynb

```diff
@@ -147,14 +141,14 @@
     "\n",
     "So I'll go with a 90% confidence interval of [3, 100].\n",
     "\n",
-    "I don't know much about El Paso, ICUs, or ventilators. I feel pretty uncertain and assign non-negligible probability to higher values. I'll go with a lognormal distribution that assigns more probability to the tails than a normal distribution."
+    "I don't know much about El Paso, ICUs, or ventilators. I feel pretty uncertain and assign non-negligible probability to higher values. I'll go with a [lognormal distribution](https://en.wikipedia.org/wiki/Log-normal_distribution), which will assign more probability to the \"right tail\" of higher values than a normal distribution."
```

Comment:
added a link to the description of lognormal on Wikipedia, which is decent

I think this notebook can definitely be accessible to someone who doesn't know what a lognormal is when they come in, and so it's good to link to explanation to make it more accessible